diff --git a/node_modules/pdfjs-dist/lib/web/ui_utils.js b/node_modules/pdfjs-dist/lib/web/ui_utils.js
index c22aae8..a16b533 100644
--- a/node_modules/pdfjs-dist/lib/web/ui_utils.js
+++ b/node_modules/pdfjs-dist/lib/web/ui_utils.js
@@ -566,6 +566,9 @@ function waitOnEventOrTimeout({
 }
 
 const animationStarted = new Promise(function (resolve) {
+  if (typeof window === 'undefined') {
+    return resolve();
+  }
   window.requestAnimationFrame(resolve);
 });
 exports.animationStarted = animationStarted;
