diff --git a/node_modules/semantic-ui-react/dist/es/modules/Popup/Popup.js b/node_modules/semantic-ui-react/dist/es/modules/Popup/Popup.js
index 9d58fd4..6c1a42c 100644
--- a/node_modules/semantic-ui-react/dist/es/modules/Popup/Popup.js
+++ b/node_modules/semantic-ui-react/dist/es/modules/Popup/Popup.js
@@ -151,7 +151,7 @@ var Popup = /*#__PURE__*/function (_Component) {
         // defined by SUI CSS. It also means that this `div` will be positioned instead of `content`.
         React.createElement("div", {
           ref: popperRef,
-          style: popperStyle
+          style: _extends({zIndex: 1900}, popperStyle)
         }, /*#__PURE__*/React.createElement(ElementType, _extends({}, contentRestProps, {
           className: classes,
           style: styles
